### DISCUSSION TITLE
Windows release binaries?

### DISCUSSION THREAD

**Discussion by gitoss** [nmoinvaz/minizip#786] [Q&A] [answered]
The last release with a Windows binary was 4.0.4 - for newer releases, I don't see any information why there is no Windows support, and no issue seems to answer this directly.

It would be nice to either have Windows binaries again, or some kind of information in the repective releases if Windows has been abandoned or what the status is - thanks!

**Comment by nmoinvaz** [comment, marked as answer]
The CI was just out of date. The next release should have Windows binaries back again.
